micrometer-core/src/test/java/io/micrometer/core/instrument/binder/mongodb/MongoMetricsConnectionPoolListenerTest.java

```diff
@@ -68,7 +73,8 @@ public void clusterOpening(ClusterOpeningEvent event) {
 
         assertThat(registry.get("mongodb.driver.pool.size").tags(tags).gauge().value()).isEqualTo(2);
         assertThat(registry.get("mongodb.driver.pool.checkedout").gauge().value()).isZero();
-        assertThat(registry.get("mongodb.driver.pool.waitqueuesize").gauge().value()).isZero();
+        // TODO: waitQueueEntered and waitQueueExited were removed, how can we provide mongodb.driver.pool.waitqueuesize?
```

Comment:
We should check why it was removed. If it isn't considered useful, then we should just note its removal in our release notes and reference Mongo changelogs/release notes/documentation explaining its removal.

Reply:
Here is [the motivation](https://jira.mongodb.org/browse/JAVA-3460) for the removal.

Reply:
I'm marking this resolved in favor of the other thread above, see my comments there: https://github.com/micrometer-metrics/micrometer/pull/2493#discussion_r593472189